### ISSUE TITLE
CPHex communication channels

### ISSUE THREAD

**Issue by guillego** [cphex/cphex#65]
Hi everyone!

It was really cool to see so many people and excitement in the last meetup!
Towards the end @vanjacosic opened the topic of the communication/coordination channels for our community and others around Copenhagen.

This thread aims to summarize some of the points that were discussed then and to open the conversation towards what channels we would like to use moving forward.

### Asynchronous messaging
Most people seemed to agree that this github repo is a good place for coordinating for meetups and discussing related topics as it's open and searchable. We can use topics and even the wiki for documenting things related to the events and workshops we organize. Let me know if there are other alternatives/proposals.

### General messaging/chat
We didn't really agree on one or the other so here are the possibilities I'm aware of.
- [Elixir Discord](https://discord.gg/elixir) -> Not sure in which channel though, couldn't find a copenhagen or denmark channel.
- [Elixir Slack](https://join.slack.com/t/elixir-lang/shared_invite/zt-1xky37xsj-9eAmbQHJ_kspA4bKBLbouQ) -> #denmark channel.

### Meetup sign-ups, attendance, advertisement
We agreed that it would make sense to merge the two meetup groups, not sure about the process.
We also discussed all the downsides of meetup as a platform but don't really have good alternatives as of right now. We commented that for the time being we can continue using github for planning and meetup for advertising the meetup and having an attendee list.

@lehoff commented that he would check with goto about using their events platform for this purposes if this meetup can fall under the scope of goto meetups.

- [Copenhagen Elixir Meetup Group](https://www.meetup.com/copenhagen-elixir-meetup-group/) -> Managed by @ErikMejerHansen.
- [Copenhagen Elixir Community](https://www.meetup.com/copenhagen-elixir-community/) -> Managed by @vanjacosic.

**Comment by Munksgaard** [comment]
> * [Elixir Slack](https://join.slack.com/t/elixir-lang/shared_invite/zt-1xky37xsj-9eAmbQHJ_kspA4bKBLbouQ) -> #denmark channel.

I'd say that a significant disadvantage here is how hard it is for outsiders to join the Elixir slack. There is no publicly available invite link and the invite bot does not work and does not seem like it will be fixed in the foreseeable future (if it even can be fixed, I'm not sure about the details).

**Comment by guillego** [comment]
@Munksgaard agree! Not sure about the details either, I have been quite happy with the amount of activity and involvement of the community in it but it's true that it's tricky to find it and join. I'm happy to post here an updated link every 20 days or so as a temporary solution.

**Comment by vanjacosic** [comment]
**Re. GitHub:**
I still don't think GitHub issues is the right platform for planning or publishing events. But since I am in the minority, I will leave it at that.

Though I would suggest that we at least enable GitHub *Discussions* and use it instead of GitHub *Issues*, as it seems like a better fit for the purpose. (See: [What is GitHub Discussions? A complete guide](https://resources.github.com/devops/process/planning/discussions/) ) Thoughts?

**Re. chat:**
I'm no big fan of Slack or Discord for public communities, they both have their issues (though I think Discord is slightly better for ease of access, history retention, search, cost etc). 

You can join the cph.community Discord, if you want. Invite link: https://discord.com/invite/NjtzmZ2APD

I've also been looking into some alternatives recently. Some are open source / self hosted, some federated, some are public and can be read without login, some look like forums or reddit etc. They all have different flaws and tradeoffs. We can discuss what kind of preferences we have at the coming meetups.

**Re. the Meetup.com groups:**
It doesn't look like Meetup allows for merging of groups. So we just have to delete one of them for now and continue looking at alternative options. Should I delete mine?

**Comment by ErikMejerHansen** [comment]
It might also make sense for us to advertise events here: https://erlef.org/events/

**Comment by ErikMejerHansen** [comment]
> It might also make sense for us to advertise events here: https://erlef.org/events/

I've signed up for a basic account at erlef.org. I'll add an event there once we have settled on the date for the next event.

**Comment by gausby** [comment]
The main problem is not the organisational aspect. Besides the point that GitHub is an organisation owned by an iffy organisation, it serves well for the organisers to plan meetings; it even serves well for the organisers to share workshop material.

The thing that doesn't work is Meetup dot com. This is evident as multiple people has created groups on there, and it is evident by the fact that I cannot find the group when I search for "Elixir" in Copenhagen. Words cannot describe how hard Meetup constantly fails meetup organisers. But unfortunately Meetup.com is accepted as the place for meetups to be discovered—and that is the main problem here:

_We have a discoverability problem, not an organising problem._

P.S. synchronous communication will not work, because it is synchronous—so Discord or, even worse, Slack will not solve any problems with organising. It will only create the problem that the organisers needs to be there—or read endless backlogs to stay up to date. No; as stupid as it may be, GitHub issues solves all the organising problems.
